subsys/bluetooth/host/Kconfig

```diff
@@ -60,6 +62,16 @@ config BT_DISCARDABLE_BUF_COUNT
 	  it will not cause the allocation for other critical events to
 	  block and may even eliminate deadlocks in some cases.
 
+config BT_DISCARDABLE_BUF_SIZE
+	int "Size of discardable event buffers"
+	range 44 255
+	default 44 if BT_OBSERVER
```

Comment:
This seems to lack a default when BT_OBSERVER is not set. IIRC we only have two events which might be considered discardable: advertising reports and inquiry responses. So you can probably make 44 the default and 255 the default when BR/EDR is enabled.